iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -254,6 +277,17 @@ public void renameTable(TableIdentifier from, TableIdentifier originalTo) {
     }
   }
 
+  private void validateTableIsIcebergTableOrView(
```

Comment:
why isn't this part of IcebergTableUtil? is it like this in upstream iceberg?

Reply:
I just copy this code from upstream iceberg.

`IcebergTableUtil `is in `iceberg-handler` module, which can not be used in `iceberg-catalog`.  

BTW, We also have similar iceberg table validate funtion in `MetaStoreUtils::isIcebergTable`. I think we can unify the validate iceberg function once the native iceberg view is integrated. This can be don in the following backport work.

Reply:
👍